doc improvement: ESB sample page (#12343)
nrfconnect/sdk-nrf: samples/esb/esb_prx/README.rst

.. _esb_prx:

Enhanced ShockBurst: Receiver
#############################

.. contents::
   :local:
   :depth: 2

The sample shows how to use the :ref:`ug_esb` protocol in receiver mode.
It shows how to configure the Enhanced ShockBurst protocol to receive packets.

Requirements
************

The sample supports the following development kits:

.. table-from-sample-yaml::

> Additionally, if you want to test the Enhanced ShockBurst Transmitter functionality, you need to build and run the :ref:`esb_prx` sample.

Reviewer: Please update the link to: _:ref:`esb_ptx`_

Successful communication is indicated by LED changes, which should be in sync on both kits.

The Receiver sample listens for packets and sends an ACK when a packet is received.
If packets are successfully received from the Transmitter, the LED pattern changes every time a packet is received.

User interface
***************

LED 1-4:
   Indicate that packets are sent or received.
   The first four packets turn on **LED 1**, **2**, **3**, and **4**.
   The next four packets turn them off again in the same order.

Configuration
*************

|config|

Building and running
********************

The Receiver sample can be found under :file:`samples/esb/esb_prx` in the |NCS| folder structure.

See :ref:`programming` for information about how to build and program the application.

FEM support
===========

.. include:: /includes/sample_fem_support.txt

Testing
=======

After programming the Transmitter sample on one of the development kits and the Receiver sample on the other kit, you can test their functionality.

.. note::

>    Follow the steps from and include the :ref:`esb_ptx` sample to test the Receiver sample.

Reviewer 2: I think this note can be removed, like the other one.

Author: Ah, missed this. Will remove.

.. _esb_prx:

Enhanced ShockBurst: Receiver
#############################

.. contents::
   :local:
   :depth: 2

The sample shows how to use the :ref:`ug_esb` protocol in receiver mode.

> It shows how to configure :ref:`ug_esb` to receive packets.

Reviewer 3: I'd leave out one of the links as they both go to the same target.

Requirements
************

The sample supports the following development kits:

.. table-from-sample-yaml::

Additionally, if you want to test the Enhanced ShockBurst Transmitter functionality, you need to build and run the :ref:`esb_prx` sample.

> You can use any two of the development kits listed above and mix different development kits.

Reviewer 3: ```suggestion
You can use any two of the listed development kits and mix different development kits.
```
In general, we should avoid using "above" or "below".

Overview
********

> The sample consists of one Receiver that use the :ref:`esb_README` library.

Reviewer 3: ```suggestion
The sample consists of one Receiver that uses the :ref:`esb_README` library.
```

> After programming each application on a nRF52 Series development kit, you can test that packets that are sent by the kit that runs the Transmitter application (by also building and running the :ref:`esb_ptx` sample) are picked up by the kit that runs the Receiver application.

Reviewer 3: ```suggestion
After programming each application on an nRF52 Series development kit, you can test that packets that are sent by the kit that runs the Transmitter application (by also building and running the :ref:`esb_ptx` sample) are picked up by the kit that runs the Receiver application.
```

Successful communication is indicated by LED changes, which should be in sync on both kits.

The Receiver example listens for packets and sends an ACK when a packet is received.

> If packets are successfully received from the transmitter, the LED pattern will change every time a packet is received.

Reviewer 3: ```suggestion
If packets are successfully received from the transmitter, the LED pattern changes every time a packet is received.
```

> You can use any two of the development kits listed above and mix different development kits to build both the current sample and the :ref:`esb_ptx` sample.

Reviewer 3: ```suggestion
You can use any two of the supported development kits and mix different development kits to build both the current sample and the :ref:`esb_ptx` sample.
```

User interface
***************

LED 1-4:
   Indicate that packets are sent or received.

>    The first four packets turn on LED 1, 2, 3, and 4.

Reviewer 3: ```suggestion
   The first four packets turn on **LED 1**, **2**, **3**, and **4**.
```

   The next four packets turn them off again in the same order.

Configuration
*************

|config|

Building and running
********************

The Receiver sample can be found under :file:`samples/esb/esb_prx` in the |NCS| folder structure.

See :ref:`programming` for information about how to build and program the application.

FEM support
===========

.. include:: /includes/sample_fem_support.txt

Testing
=======

After programming the Transmitter sample on one of the development kits and the Receiver sample on the other kit, you can test their functionality.

.. note::
   Follow the steps from and include the :ref:`esb_ptx` sample to test the Receiver sample.

Complete the following steps to test both the Transmitter and Receiver samples:

1. Power on both kits.
#. Observe that the LEDs change synchronously on both kits.
#. Optionally, connect to the kits with a terminal emulator (for example, PuTTY).
   See :ref:`putty` for the required settings.
#. Observe the logging output for both kits.

Dependencies
************

> This sample uses the following |NCS| libraries:

Reviewer 3: ```suggestion
This sample uses the following |NCS| library:
```
